src/wmnds/components/textarea/_textarea.njk

```diff
@@ -0,0 +1,64 @@
+{% macro wmndsTextarea(params) %}
+
+{# Imports #}
+{% from "wmnds/components/form-elements/label/_label.njk" import wmndsLabel %}
+{% from "wmnds/components/form-elements/error-message/_error-message.njk" import wmndsFeErrorMessage %}
+
+{# Params #}
+{% set id = params.id if params.id else "textarea" %}
+{% set name = params.name if params.name else id %}
+{% set rows = params.rows if params.rows else 3 %}
+{% set required = params.required if params.required else false %}
+{% set classes = " " + params.classes if params.classes %}
+{# Error Message Params #}
+{% set groupErrorClass = " wmnds-fe-group--error" if params.error %}
+{% set textareaErrorClass = " wmnds-fe-input--error" if params.error %}
+{% set errorContentText = params.errorMessage.contentText if params.errorMessage.contentText else null %}
```

Comment:
```suggestion
{% set errorContentText = params.errorMessage.contentText if params.errorMessage.contentText else "Please enter some text" %}
```

Reply:
I'll commit it but it has a fallback inside the errorMessage component. 👍  but that way is a bit more customized for textarea. :)